store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -532,106 +567,161 @@ private boolean validateBoolean(String param, String value) throws ParseExceptio
+        printOptWithDesc(O_PP,  8,  H_DESC_VOL_PP,        true);
+        printOptWithDesc(O_AP,  8,  H_DESC_VOL_AP,        true);
+        printOptWithDesc(O_ACC, 8,  H_DESC_VOL_ACC,       true);
+        printOptWithDesc(O_URL, 8,  H_DESC_VOL_URL,       true);
+        printOptWithDesc(O_NS,  8,  H_DESC_VOL_NS,        true);
+        printOptWithDesc(O_P,   8,  H_DESC_VOL_RP,        true);
+        printOptWithDesc(O_VP,  8,  H_DESC_VOL_PRE,       true);
+        printOptWithDesc(O_SMC, 8,  H_DESC_VOL_SMC,       false);
+        printLineWithLeftPad(H_NOTE_EXT_OI_VOL_ADD, 4);
+        printLineWithLeftPad(H_EXP_EXT_OI_VOL_ADD, 4);
+
+        printLineWithLeftPad(STR_EMPTY, 0);
+        printLineWithLeftPad(H_TLE_INT_VOL_EDIT, 0);
+        printOptWithDesc(O_E,   4,  H_DESC_VOL_EDIT,      null);
+        printOptWithDesc(O_ID,  8,  H_DESC_VOL_ID,        true);
+        printOptWithDesc(O_N,   8,  H_DESC_VOL_NAME,      true);
+        printOptWithDesc(O_T,   8,  H_DESC_VOL_TYPE_PSI,  true);
+        printOptWithDesc(O_P,   8,  H_DESC_VOL_RP,        true);
+        printOptWithDesc(O_C,   8,  H_DESC_VOL_CB,        false);
+        printOptWithDesc(O_CT,  8,  H_DESC_VOL_CT,        false);
+        printLineWithLeftPad(H_EXP_INT_VOL_EDIT, 4);
+
+        printLineWithLeftPad(STR_EMPTY, 0);
+        printLineWithLeftPad(H_TLE_EXT_VOL_EDIT, 0);
+        printOptWithDesc(O_E,   4,  H_DESC_VOL_EDIT,      null);
+        printOptWithDesc(O_ID,  8,  H_DESC_VOL_ID,        true);
+        printOptWithDesc(O_N,   8,  H_DESC_VOL_NAME,      true);
+        printLineWithLeftPad(H_NOTE_EXT_VOL_EDIT, 4);
+        printLineWithLeftPad(H_EXP_EXT_VOL_EDIT, 4);
+
+        printLineWithLeftPad(STR_EMPTY, 0);
+        printLineWithLeftPad(H_TLE_VOL_DEL, 0);
+        printOptWithDesc(O_D,   4,  H_DESC_VOL_DEL,       null);
+        printOptWithDesc(O_ID,  8,  H_DESC_VOL_ID,        true);
+        printLineWithLeftPad(H_EXP_VOL_DEL, 4);
+
+        printLineWithLeftPad(STR_EMPTY, 0);
+        printLineWithLeftPad(H_TLE_OTHER_VOL_OPTS, 0);
+        printOptWithDesc(O_L,   4,  H_DESC_VOL_LST,       null);
+        printOptWithDesc(O_ID,  8,  H_DESC_VOL_ID,        true);
+        printOptWithDesc(O_DC,  4,  H_DESC_VOL_DISP_CURR, null);
+        printOptWithDesc(O_TS,  4,  H_DESC_VOL_SET_CURR,  null);
+        printOptWithDesc(O_TS,  4,  H_DESC_VOL_OFF_CURR,  null);
+        printLineWithLeftPad(STR_EMPTY, 0);
     }
 
-    private void printOpt(String optStr, int leftPad) {
+    private void printOptWithDesc(String optStr, int leftPad, String desc, Boolean isRequired) {
         Options options = getOptions();
         Option opt = options.getOption(optStr);
         StringBuilder buf = new StringBuilder();
         buf.append(Strings.repeat(" ", leftPad));
         buf.append('-').append(opt.getOpt()).append(",--").append(opt.getLongOpt());
-        if (opt.hasArg()) {
-            buf.append(" <arg>");
+        if (opt.hasArg() && null != isRequired) {
```

Comment:
May be your `opt.hasArg()` should be enough here.

Reply:
Done